default_gateway.go

```diff
@@ -52,6 +53,16 @@ func (sb *sandbox) setupDefaultGW() error {
 		eplen = len(sb.containerID)
 	}
 
+	sbLabels := sb.Labels()
+
+	if sbLabels[netlabel.PortMap] != nil {
+		createOptions = append(createOptions, CreateOptionPortMapping(sbLabels[netlabel.PortMap].([]types.PortBinding)))
+	}
+
+	if sbLabels[netlabel.ExposedPorts] != nil {
+		createOptions = append(createOptions, CreateOptionExposedPorts(sbLabels[netlabel.ExposedPorts].([]types.TransportPort)))
+	}
+
```

Comment:
From my understanding it looks like the external connectivity cannot be programmed after the endpoint has been created on the windows driver.
So here they are bringing back the old model of passing the exposedports/port mappings during the default gw endpoint creation.